tests/checker/bad/gold/libdup2.scilla.gold

```diff
@@ -1,13 +1,12 @@
 {
-  "gas_remaining": "7834",
+  "gas_remaining": "8000",
   "errors": [
     {
-      "error_message":
-        "Entry sort_uint32 in library TestLib3 conflicts with entry in library TestLib1",
+      "error_message": "Variable sort_uint32 imported from multiple sources",
```

Comment:
From a  private discussion: it is possible to keep the error message more informative at least for offline development.

Reply:
See https://github.com/Zilliqa/scilla/pull/905/commits/2836148c25c7ab18f18dc91f8835b9f1908d6dbd

It's not an ideal way to generate error messages, but I don't want to copy the code that performs the duplicate entry check.